src/UnitTest/test_quote_prov.cpp

```diff
@@ -1,709 +1,876 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Do you know why this is showing up as a bulk delete then add? It makes it really hard to review... is it a line ending change or something? Can you change it back so we can review just the lines which have changed?